tools/integration_tests/static_mounting/static_mounting.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2023 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package static_mounting
```

Comment:
isn't it exactly same as what setup.go was doing ? If yes, we do we need it separately ?

Reply:
I make only common things in the setup.
so now if we want to mount gcsfuse statically - we will set all arguments accordingly and call setup.MountGcsfuse() to run with that arguments
same for only dir
and upcoming dynamic mounting, persistent mounting